### PR TITLE
feat: [PL-21390]: Hierarchical Resource Groups and Permissions

### PR DESCRIPTION
**Pull request by reetika04**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by reetika04** [comment]
Screenshot 2022-01-21 at 2 26 46 PM
Screenshot 2022-01-21 at 2 26 55 PM

**Comment by reetika04** [comment]
Screenshot 2022-01-21 at 3 02 33 PM

**Comment by reetika04** [comment]
retrigger jest

File: src/modules/20-rbac/pages/ResourceGroupDetails/utils.tsx [hunk 1]
```diff
@@ -128,3 +157,118 @@ export const computeResourceMapOnMultiChange = (
...
+  scope: Scope,
+  resourceSelectors?: ResourceSelector[]
+): string => {
+  const selectorScope = getScopeForResourceGroup(resourceSelectors)
+  const dropDownItems = getScopeDropDownItems(scope, getString)
+  const option = dropDownItems.filter(item => item.value === selectorScope)
+  return option.length ? option[0].label : ''
+}
+
+export const getScopeDropDownItems = (scope: Scope, getString: UseStringsReturn['getString']): SelectOption[] => {
```
Comment (abhinavrastogi-harness) [inline src/modules/20-rbac/pages/ResourceGroupDetails/utils.tsx:176]: tests added for newly added utils?

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by ghost — DISMISSED** [review]

**Comment by reetika04** [comment]
trigger cypress

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by karansiwach360 — APPROVED** [review]

**Review by swarajcheguri-harness — APPROVED** [review]

**Pull request merged**